python/orca/src/bigdl/orca/learn/pytorch/pytorch_ray_estimator.py

```diff
@@ -416,6 +416,28 @@ def load(self, model_path):
         state_dict = torch.load(model_path)
         self.load_state_dict(state_dict)
 
+    def save_checkpoint(self, model_path):
+        from bigdl.dllib.utils.file_utils import is_local_path
+        if is_local_path(model_path):
+            self.save(model_path)
+        else:
+            results = [
+                worker.save_checkpoint.remote(model_path)
```

Comment:
why saving `checkpoint` in every worker?

Does it work if multiple workers in the same node saving to the same location?

Reply:
`worker.save_checkpoint` is responsible for correctly handling the behavior in distributed training, which saves checkpoint only on rank 0. (In that way, `ModelCheckpoint` is transparent for both local and ddp).
In that way, it does work for multiple workers on the same node.